osu.Game/Rulesets/UI/DrawableRuleset.cs

```diff
@@ -184,11 +184,13 @@ private void load(CancellationToken? cancellationToken)
                     {
                         RelativeSizeAxes = Axes.Both,
                         Child = KeyBindingInputManager
-                            .WithChild(CreatePlayfieldAdjustmentContainer()
-                                .WithChild(Playfield)
-                            ),
+                            .WithChildren(new Drawable[]
+                            {
+                                CreatePlayfieldAdjustmentContainer()
+                                    .WithChild(Playfield),
+                                Overlays
```

Comment:
As per CI, this breaks `ModMuted`. See this part:

https://github.com/ppy/osu/blob/455fb7789ccf70949cfa21b6222a1838bf7f56fe/osu.Game/Rulesets/Mods/ModMuted.cs#L64-L77

If `AffectsHitSounds` is true, then the mod mutes the `AudioContainer`. The mod also plays metronome, which is added to `Overlays`. Previously this wasn't an issue because the two containers were siblings - but now `Overlays` is a child of the `AudioContainer`, ergo the failure.

Reply:
It's always a very happy day when tests fail and reveal an issue we definitely would have missed otherwise.

Reply:
I've fixed this by updating `ModMuted` to add to `FrameStableComponents` instead of `Overlays`, which actually feels more correct as the metronome is a component and not an overlay.